### PR TITLE
Move copy as CSV button to be a right-click option

### PR DESCRIPTION
**Pull request by vadi2**
#### Brief overview of PR changes/additions
Don't swap the 'open new resource' button for a 'copy results as CSV' anymore - it's confusing to change meaning of the same location like that. Instead, add the 'copy results as CSV' to the status box indicators.
#### Motivation for adding to Hammer
Fix https://github.com/health-validator/Hammer/issues/82
#### Other info (issues closed, discussion etc)
Once https://github.com/health-validator/Hammer/pull/71 is added, I'll add the right-click option to the results list as well. I think the feature will be even more discoverable there.

### CODE REVIEW

File: Main.qml [hunk 1]
```diff
@@ -307,6 +292,18 @@ ApplicationWindow {
...
+        Menu {
+            id: contextMenu
+            MenuItem {
+                text: qsTr("Copy all as CSV")
+                onTriggered: { appmodel.copyValidationReport(); toast.show(qsTr("Copied all results as a CSV")) }
+            }
+        }
+
+        function openContextMenu() {
+            contextMenu.open()
```
Comment (vadi2) [inline Main.qml]: It seems passing the `x,y` parameters is optional - it defaults to the current mouse position.

**Comment by vadi2** [comment]
Ah - it had to go because it allow the 'open new resource' button to stay. Otherwise, it wasn't very intuitive how to open a new resource, now that the back button morphs into revalidate.

**Comment by pieter-edelman-nictiz** [comment]
I get that, but there's more than enough room for both buttons, is it not? Anyway, no big issue, just wondering.

**Comment by vadi2** [comment]
I don't want to have a row of too many buttons, I think it won't be as sleek then. What do you think?

**Comment by pieter-edelman-nictiz** [comment]
TBH I think it's a good idea to take a good look at the UI in general regarding the things outside 'the normal flow' at some point in time as the feature set grows. But I wouldn't stress it for now, the proposed solution is totally fine by me.

**Comment by vadi2** [comment]
For sure.

**Pull request merged**